datasets.py

```diff
@@ -702,4 +702,148 @@ def from_path(self, file_path):
         else:
             anns = [None, [0], [country], file_path, None]
 
-        return image, anns
\ No newline at end of file
+
+        return image, anns
+
+#Works on celebA image dataset of celebrities
```

Comment:
maybe just say "CelebA face dataset" and then link to where the images came from